### PR TITLE
Various fixes for inlines

### PR DESCRIPTION
**Pull request by farridav**
- Add in cli tool for generating diffs between django template overides and originals
- Rewrite tabular in-lines, bring back to django template, add minor changes as necessary
- Fixup stacked inlines too, namely:
  - Fixes showing hidden form
  - Fixes extra #num is not incrementing properly

@TysonRV let me know if this Fixes https://github.com/farridav/django-jazzmin/pull/165 and https://github.com/farridav/django-jazzmin/issues/125

### CODE REVIEW

File: cli.py [hunk 1]
```diff
@@ -42,12 +46,46 @@ def locales(cmd_args):
...
-    parser_locales = subparsers.add_parser('locales', help='remove the django provided strings')
-    parser_locales.add_argument("--locale", action="store", dest="locale", help="locale to process", default="de")
+    parser_locales = subparsers.add_parser("locales", help="remove the django provided strings")
+    parser_locales.add_argument("--prune", action="store", dest="locale", help="locale to prune", default="de")
     parser_locales.set_defaults(func=locales)
+    parser_templates = subparsers.add_parser("templates", help="Deal with templates")
+    parser_templates.add_argument("--diff", action="store_true", dest="template_diff", help="generate template diff")
+    parser_templates.set_defaults(func=templates)
     cmd_args = parser.parse_args()
     cmd_args.func(cmd_args)
```
Comment (TysonRV) [inline cli.py:90]: This works quite well with the right editor. Great addition!

File: jazzmin/static/jazzmin/css/django.css [hunk 2]
```diff
@@ -8,8 +8,12 @@ div.inline-related {
     padding: 5px;
 }
 
-.inlinechangelink {
-    content: url('../img/icon-changelink.svg');
+.table tr.form-row {
+    display: table-row;
+}
```
Comment (TysonRV) [inline jazzmin/static/jazzmin/css/django.css:13]: Much more elegant than my solution really

File: jazzmin/templates/admin/edit_inline/stacked.html [hunk 3]
```diff
@@ -6,28 +6,33 @@
...
                                             {% if inline_admin_formset.has_change_permission %}
-                                                <i class="fas fa-pencil-alt fa-sm float-right"></i>
+                                                <i class="fas fa-pencil-alt fa-sm"> </i>
                                             {% else %}
-                                                <i class="fas fa-eye fa-sm float-right"></i>
+                                                <i class="fas fa-eye fa-sm"> </i>
                                             {% endif %}
                                             </a>
-                                        </span>
+                                            {{ inline_admin_form.original }}
```
Comment (TysonRV) [inline jazzmin/templates/admin/edit_inline/stacked.html:25]: This makes the title line disappear when `show_change_link = False`

File: jazzmin/templates/admin/edit_inline/stacked.html [hunk 4]
```diff
@@ -6,28 +6,33 @@
...
                                     {% endif %}
-                                    {{ inline_admin_form.original }}
                                 {% else %}
-                                <span class="text-success"><i class="fas fa-plus fa-sm"></i></span>  {% trans "Add new " %}{{ inline_admin_formset.opts.verbose_name|capfirst }} - #{{ forloop.counter }}
+                                    <i class="fas fa-plus fa-sm text-success"> </i>
+                                    {% trans "New" %} {{ inline_admin_formset.opts.verbose_name|capfirst }}
```
Comment (TysonRV) [inline jazzmin/templates/admin/edit_inline/stacked.html:29]: Much better, so we don't have to add another translation

File: jazzmin/templates/admin/edit_inline/tabular.html [hunk 5]
```diff
@@ -1,63 +1,46 @@
...
-                                        <a href="{% url inline_admin_form.model_admin.opts|admin_urlname:'change' inline_admin_form.original.pk|admin_urlquote %}" class="{% if inline_admin_formset.has_change_permission %}inlinechangelink{% else %}inlineviewlink{% endif %}">{% if inline_admin_formset.has_change_permission %}{% trans "Change" %}{% else %}{% trans "View" %}{% endif %}</a>
-                                    {% endif %}
-                                {% endif %}
-                                {% if inline_admin_form.show_url %}
-                                    <a href="{{ inline_admin_form.absolute_url }}">{% trans "View on site" %}</a>
+                                    {% if inline_admin_form.model_admin.show_change_link and inline_admin_form.model_admin.has_registered_model %}<a href="{% url inline_admin_form.model_admin.opts|admin_urlname:'change' inline_admin_form.original.pk|admin_urlquote %}" class="{% if inline_admin_formset.has_change_permission %}inlinechangelink{% else %}inlineviewlink{% endif %}">{% if inline_admin_formset.has_change_permission %}{% trans "Change" %}{% else %}{% trans "View" %}{% endif %}</a>{% endif %}
```
Comment (TysonRV) [inline jazzmin/templates/admin/edit_inline/tabular.html:33]: Wouldn't it be better to add the icons as opposed to the `Change` and `View` links? It doesn't look too jazzy this way

**Review by TysonRV — CHANGES_REQUESTED** [review]
@farridav Great job here!

Needs a couple of small changes and wanted to know your opinion on the screenshot tabulars. I think we could replace that dull link with an amazing pencil icon 😄  ...and an eye for the view one. That way we also keep things consistent

Screenshot 2020-10-06 at 17 47 16

File: jazzmin/templates/admin/edit_inline/stacked.html (hunk 3 above)
Comment (farridav) [reply jazzmin/templates/admin/edit_inline/stacked.html:25]: aah i had wondered why you moved it out

**Comment by farridav** [comment]
@TysonRV great idea ! ive updated the PR, hopefully this gets us where we need to be, but feel free to add more jazz on top if needed ;)

File: jazzmin/templates/admin/edit_inline/tabular.html (hunk 5 above)
Comment (farridav) [reply jazzmin/templates/admin/edit_inline/tabular.html:33]: done

**Review by TysonRV — APPROVED** [review]

**Pull request merged**